### PR TITLE
feat(docs): document `telegram` package

### PR DESCRIPTION
**Pull request by pierwill**

### CODE REVIEW

File: stdlib/contrib/sranka/telegram/telegram.flux [hunk 1]
```diff
@@ -39,13 +99,72 @@ message = (
...
+//   |> endpoint(mapFn: (r) => ({
+//       channel: "-12345",
+//       text: "Disk usage is **${r.status}**.",
+//       silent: true
+//     })
+//   )()
+// ```
+// 
+// tag: notification-endpoints
 endpoint = (url=defaultURL, token, parseMode=defaultParseMode, disableWebPagePreview=defaultDisableWebPagePreview) => (mapFn) => (tables=<-) => tables
```
Comment (pierwill) [inline stdlib/contrib/sranka/telegram/telegram.flux:168]: I would expect the formatter to split this line.

File: stdlib/contrib/sranka/telegram/telegram.flux [hunk 2]
```diff
@@ -1,22 +1,82 @@
+// Package telegram provides functions for sending messages to [Telegram](https://telegram.org/)
+// using the [Telegram Bot API](https://core.telegram.org/bots/api).
+//
+// FIXME:Include this content?
+// https://docs.influxdata.com/flux/v0.x/stdlib/contrib/sranka/telegram/#set-up-a-telegram-bot
```
Comment (sanderson) [inline stdlib/contrib/sranka/telegram/telegram.flux:5]: Yes, we should include this content.

File: stdlib/contrib/sranka/telegram/telegram.flux [hunk 3]
```diff
@@ -1,22 +1,82 @@
...
+// telegram.message(
+//   url: "https://api.telegram.org/bot",
+//   token: "S3crEtTel3gRamT0k3n",
+//   channel: "-12345",
+//   text: "Example message text",
+//   parseMode: "MarkdownV2",
+//   disableWebPagePreview: false,
+//   silent: true
+// )
+// ```
```
Comment (sanderson) [inline stdlib/contrib/sranka/telegram/telegram.flux:41]:
~~~suggestion
~~~

File: stdlib/contrib/sranka/telegram/telegram.flux [hunk 4]
```diff
@@ -1,22 +1,82 @@
...
+//
+// ## Parameters
+// 
+// - url: URL of the Telegram bot endpoint. Default is `https://api.telegram.org/bot`.
+// - token: (Required) Telegram bot token.
+// - channel: (Required) Telegram channel ID.
```
Comment (sanderson) [inline stdlib/contrib/sranka/telegram/telegram.flux:47]:
```suggestion
// - token: Telegram bot token.
// - channel: Telegram channel ID.
```

File: stdlib/contrib/sranka/telegram/telegram.flux [hunk 5]
```diff
@@ -1,22 +1,82 @@
...
+//     |> filter(fn: (r) => r._measurement == "statuses")
+//     |> last()
+//     |> findRecord(fn: (key) => true, idx: 0)
+// 
+//     telegram.message(
+//       token: token,
+//       channel: "-12345"
+//       text: "Disk usage is **${lastReported.status}**.",
+//     )
+// ```
```
Comment (sanderson) [inline stdlib/contrib/sranka/telegram/telegram.flux:79]:
~~~suggestion
// ```no_run
// import "influxdata/influxdb/secrets"
// import "contrib/sranka/telegram"
//
// token = secrets.get(key: "TELEGRAM_TOKEN")
//
// lastReported = from(bucket: "example-bucket")
//     |> range(start: -1m)
//     |> filter(fn: (r) => r._measurement == "statuses")
//     |> last()
//     |> findRecord(fn: (key) => true, idx: 0)
//
// telegram.message(
//     token: token,
//     channel: "-12345",
//     text: "Disk usage is **${lastReported.status}**.",
// )
// ```
//
// tags: single notification
//
~~~

File: stdlib/contrib/sranka/telegram/telegram.flux [hunk 6]
```diff
@@ -39,13 +99,72 @@ message = (
...
+// import "contrib/sranka/telegram"
+// 
+// telegram.endpoint(
+//   url: "https://api.telegram.org/bot",
+//   token: "S3crEtTel3gRamT0k3n",
+//   parseMode: "MarkdownV2",
+//   disableWebPagePreview: false,
+// )
+// ```
```
Comment (sanderson) [inline stdlib/contrib/sranka/telegram/telegram.flux:113]:
```suggestion
```

File: stdlib/contrib/sranka/telegram/telegram.flux [hunk 7]
```diff
@@ -39,13 +99,72 @@ message = (
...
+// - `channel`
+// - `text`
+// - `silent`
+// 
+// For more information, see `telegram.message()` parameters.
+//
+// ## Parameters
+// 
+// - url: URL of the Telegram bot endpoint. Default is `https://api.telegram.org/bot`.
+// - token: (Required) Telegram bot token.
```
Comment (sanderson) [inline stdlib/contrib/sranka/telegram/telegram.flux:134]:
```suggestion
// - token: Telegram bot token.
```

File: stdlib/contrib/sranka/telegram/telegram.flux [hunk 8]
```diff
@@ -39,13 +99,72 @@ message = (
...
+// - parseMode: [Parse mode](https://core.telegram.org/bots/api#formatting-options)
+//   of the message text.
+//   Default is `MarkdownV2`.
+// - disableWebPagePreview: Disable preview of web links in the sent message.
+//   Default is false.
+// 
+//   The returned factory function accepts a `mapFn` parameter.
+//   The `mapFn` must return an object with `channel`, `text`, and `silent`,
+//   as defined in the `message` function arguments.
```
Comment (sanderson) [inline stdlib/contrib/sranka/telegram/telegram.flux:143]:
```suggestion
// The returned factory function accepts a `mapFn` parameter.
// The `mapFn` must return an record with the following properties:
//
// - `channel`
// - `text`
// - `silent`
//
// See `telegram.message` parameters for more information.
```

File: stdlib/contrib/sranka/telegram/telegram.flux [hunk 9]
```diff
@@ -39,13 +99,72 @@ message = (
...
+// crit_statuses
+//   |> endpoint(mapFn: (r) => ({
+//       channel: "-12345",
+//       text: "Disk usage is **${r.status}**.",
+//       silent: true
+//     })
+//   )()
+// ```
+// 
+// tag: notification-endpoints
```
Comment (sanderson) [inline stdlib/contrib/sranka/telegram/telegram.flux:167]:
~~~suggestion
// ```no_run
import "influxdata/influxdb/secrets"
// import "contrib/sranka/telegram"

// token = secrets.get(key: "TELEGRAM_TOKEN")
// endpoint = telegram.endpoint(token: token)

// crit_statuses = from(bucket: "example-bucket")
//     |> range(start: -1m)
//     |> filter(fn: (r) => r._measurement == "statuses" and status == "crit")

// crit_statuses
//     |> endpoint(
//         mapFn: (r) => ({
//             channel: "-12345",
//             text: "Disk usage is **${r.status}**.",
//             silent: true,
//         }),
//     )()
// ```
// 
// tag: notification endpoints
~~~

File: stdlib/contrib/sranka/telegram/telegram.flux (hunk 1 above)
Comment (sanderson) [reply stdlib/contrib/sranka/telegram/telegram.flux:168]: From what I've seen, the formatter likes arrow operations (`=>`) to be on a single line.

**Review by sanderson — CHANGES_REQUESTED** [review]

**Review by sanderson — APPROVED** [review]

**Review by rockstar — APPROVED** [review]

**Pull request merged**